pb/client/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"foxygo.at/pony/pb/echo/v1"
+	"github.com/alecthomas/kong"
+	"google.golang.org/grpc"
+)
+
+var version = "v0.0.0"
+
+type config struct {
+	Address string           `help:"hostname:port" default:"localhost:9090"`
+	Message string           `arg:"" help: "message to send" default:"Hello 🌏"`
```

Comment:
the space after `help: ` is causing a lint error.

Reply:
done.